ui/components/multichain/pages/review-permissions-page/review-permissions-page.tsx

```diff
@@ -0,0 +1,204 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory, useParams } from 'react-router-dom';
+import { NonEmptyArray } from '@metamask/utils';
+import {
+  AlignItems,
+  BackgroundColor,
+  BlockSize,
+  Display,
+  FlexDirection,
+  IconColor,
+  JustifyContent,
+  TextAlign,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import { getURLHost } from '../../../../helpers/utils/util';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import {
+  getConnectedSitesList,
+  getNonTestNetworks,
+  getOrderedConnectedAccountsForConnectedDapp,
+  getPermissionSubjects,
+  getPermittedChainsForSelectedTab,
+} from '../../../../selectors';
+import { removePermissionsFor } from '../../../../store/actions';
+import {
+  AvatarFavicon,
+  AvatarFaviconSize,
+  Box,
+  Button,
+  ButtonIcon,
+  ButtonIconSize,
+  ButtonSize,
+  ButtonVariant,
+  Icon,
+  IconName,
+  IconSize,
+  Text,
+} from '../../../component-library';
+import { ToastContainer, Toast } from '../..';
+import { NoConnectionContent } from '../connections/components/no-connection';
+import { Content, Footer, Header, Page } from '../page';
+import { SubjectsType } from '../connections/components/connections.types';
+import { SiteCell } from '.';
+
+export const ReviewPermissions = () => {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const urlParams: { origin: string } = useParams();
+  const securedOrigin = decodeURIComponent(urlParams.origin);
+  const [showAccountToast, setShowAccountToast] = useState(false);
+  const [showNetworkToast, setShowNetworkToast] = useState(false);
+  const activeTabOrigin: string = securedOrigin;
+  const subjectMetadata: { [key: string]: any } = useSelector(
+    getConnectedSitesList,
+  );
+  const connectedSubjectsMetadata = subjectMetadata[activeTabOrigin];
+  const connectedNetworks = useSelector((state) =>
+    getPermittedChainsForSelectedTab(state, activeTabOrigin),
+  );
+  const networksList = useSelector(getNonTestNetworks);
+  const connectedAccounts = useSelector((state) =>
+    getOrderedConnectedAccountsForConnectedDapp(state, activeTabOrigin),
+  );
+  const subjects = useSelector(getPermissionSubjects);
+  const grantedNetworks = networksList.filter(
+    (net: { chainId: any }) => connectedNetworks.indexOf(net.chainId) !== -1,
+  );
+  const hostName = getURLHost(securedOrigin);
+  const disconnectAllAccounts = () => {
```

Comment:
This method is also present in `edit-accounts-modal.tsx`, line 93

Reply:
It's removed from edit-accounts-modal.tsx now. Passed via this component now